lua/lspconfig/stylelint_lsp.lua

```diff
@@ -0,0 +1,37 @@
+local configs = require('lspconfig/configs')
+local util = require('lspconfig/util')
+
+configs.stylelint_lsp = {
+  default_config = {
+    cmd = {'stylelint-lsp', '--stdio'},
+    filetypes = {
+      'css',
+      'less',
+      'postcss',
```

Comment:
When is the postcss filetype used? I thought that was just post-processing of css.

Reply:
I assume dewey92 copied that from me ([coc-stylelintplus](https://github.com/bmatcuk/coc-stylelintplus)), and I copied that from somewhere else - though, I can't remember where. Perhaps coc-stylelint?

Reply:
> I assume dewey92 copied that from me (coc-stylelintplus)

💯 I've never worked with CSS (and its pre-processors) files in vim as I use CSS-in-JS on a day to day basis. So not sure if `postcss` is a valid filetype

Reply:
All I could find was this which is almost 5 years old marked as "work in progress", which makes me think it's abandoned: https://github.com/stephenway/postcss.vim

Maybe skip it for now until someone actually needs it?

Reply:
Yup. Done :)